src/app/app.component.html

```diff
@@ -0,0 +1,519 @@
+<style>
```

Comment:
clear this file completely. This is all generated content. Just include <div>ecommerce site</div> for now